packages/artillery/lib/cmds/report.js

```diff
@@ -35,6 +36,21 @@ ReportCommand.flags = {
   })
 };
 
+const deprecationNotice = `
+\x1b[34m┌───────────────────────────────────────────────────────────────────────┐
```

Comment:
We should use `chalk` here (already used by the CLI) which e.g. takes care of checking if output is being redirected in which case ANSI codes are not included in the output.